scripts/install_ddev_wsl2_docker_desktop.ps1

```diff
@@ -7,7 +7,7 @@
 # Run this in an administrative PowerShell window.
 # You can download, inspect, and run this, or run it directly with
 # Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072;
-# iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/master/scripts/install_ddev_wsl2_docker_desktop.ps1'))
+# iex ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/ddev/ddev/main/scripts/install_ddev_wsl2_docker_desktop.ps1'))
```

Comment:
These are the class of things we'll want to manually test after the change. GitHub should do redirection for us, but if it messes up then people will have errors with all the scripts between the time the branch name change happens and the next release.

Reply:
There is no redirect, it opens files from `https://raw.githubusercontent.com` just fine as before.

Reply:
Yes, that's what I meant too. The URL with `master` still opens as before.

I'm curious whether the `master` link will stay in sync with `main` or if it will remain fixed at the point when the `master` branch was last used. We'll see after this PR is merged and the script content is updated.

Reply:
That's a good point and super important, thanks for thinking ahead. I imagine and hope it will stay in sync.

Reply:
It stays in sync 👍